src/interfaces/ILineaWorldID.sol

```diff
@@ -2,7 +2,7 @@
 pragma solidity ^0.8.15;
 
 /// @title Interface for the LineaWorldID contract
-/// @author Worldcoin and 0xprinc
+/// @author Worldcoin and 0xprinc x James Harrison.
 /// @notice Interface for the CrossDomainOwnable contract for the Linea L2
```

Comment:
Since we don't have CrossDomainOwnable implementation for Linea (yet?) and use raw MessageService contract, let's call it ... MessageServiceBase contract (because it inherits from `MessageServiceBase`? Or maybe you have better options